plasmapy/examples/plot_braginskii.py

```diff
@@ -0,0 +1,51 @@
+# coding: utf-8
+"""
+Braginskii coefficients
+=========================
+
+A short example of how to calculate classical transport coefficients
+from Bragiński's theory.
+"""
+
+from astropy import units as u
+from plasmapy import physics
+import matplotlib.pyplot as plt
+import numpy as np
+
+#####################################################
+# We'll use some sample ITER data, without much regard for whether
+# the regime is even fit for classical transport theory:
+
+electron_temperature = 8.8 * u.keV
```

Comment:
While https://github.com/astropy/astropy/issues/6999 is unresolved, we should be careful about variable names so that the units are consistent with them.  Right now, electron_temperature has units of energy, which could lead to unexpected behavior since someone might think it is in units of temperature.  My suggestion would be to either convert it to kelvin here using the appropriate equivalency...
```Python
electron_temperature = (8.8 * u.keV).to(u.K, equivalencies=u.temperature_energy())
```
...or change the variable name so that it reflects that it is thermal energy per particle (maybe `themal_energy_per_electron` or something).

Reply:
I've been thinking about them equivalencies a bit lately. I might have some other solutions to that a bit later. Just leaving this note here for now.

Reply:
I changed the variable names!